build-support/githooks/pre-commit

```diff
@@ -38,6 +38,3 @@ fi
 
 echo "* Checking __init__.py files"
 $_PANTS_PRE_COMMIT_BINARY run build-support/bin/check_inits.py
-
-echo "* Checking for banned imports"
-$_PANTS_PRE_COMMIT_BINARY run build-support/bin/check_banned_imports.py
```

Comment:
Not running these checks during pre-commit no more?

Reply:
:man_shrugging: We could `pants test path/to:this_test` but do we get _that_ much value?

If the remediation is just swapping `use`s I don't think it's fatal to not report that until the user is testing their code.

I'd rather we do _less_ custom things :smile:  (open for discussion though)

Reply:
Tell you what, let me add a comment to switch to `lint` when we do https://github.com/pantsbuild/pants/issues/17729

Reply:
Better?

Reply:
sounds good to me :)